samples/modules/canopennode/src/main.c

```diff
@@ -77,28 +64,16 @@ static void led_callback(bool value, void *arg)
  */
 static void config_leds(CO_NMT_t *nmt)
 {
-#ifdef LED_GREEN_PORT
-	led_green.dev = device_get_binding(LED_GREEN_PORT);
-	led_green.pin = LED_GREEN_PIN;
-	if (led_green.dev) {
-		gpio_pin_configure(led_green.dev, LED_GREEN_PIN,
-				   GPIO_OUTPUT_INACTIVE
-				   | LED_GREEN_FLAGS);
+	if (led_green_gpio.port) {
```

Comment:
You need a call to `device_is_ready()` before calling `gpio_pin_configure()` for both LEDs.

Reply:
Right! Added a logic and messages for all cases and checked for `gpio_pin_configure_dt` return value as well. Maybe overkill but since it's a sample it may catch some corner case.